usagov/usagov-2021
# Usagov 1958 document analytics

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-19958

## Description

Update the links in the life events grid to have the same data attribute for analytics that the search results do.
Documents the data attributes as CSS selectors to be handled with care.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements


- [x] Requires New Config Updated view

### Validation Steps

- Load the benefit search page, inspect the titles for life events in the grid above the search results. The `<a>` links should have a `data-analytics="life-events-links"` attribute

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
